client/me/purchases/cancel-purchase/index.jsx

```diff
@@ -222,7 +215,7 @@ class CancelPurchase extends React.Component {
 						purchase={ purchase }
 						includedDomainPurchase={ this.props.includedDomainPurchase }
 						disabled={ this.state.cancelBundledDomain && ! this.state.confirmCancelBundledDomain }
-						selectedSite={ this.props.selectedSite }
+						siteSlug={ this.props.siteSlug }
```

Comment:
It looks like `selectedSite` is still used by the `CancelPurchaseButton` component (when it's available) so don't we need to pass both `selectedSite` and `siteSlug` in here, rather than just the latter?

Reply:
Yep, you are correct. Thanks, I also missed that when refactoring back in `selectedSite`.